web/impact/impact/tests/contexts/user_context.py

```diff
@@ -44,4 +48,9 @@ def __init__(self,
         elif user_type == "MEMBER":
             self.profile = MemberProfileFactory(user=self.user)
             user.memberprofile = self.profile
+        if program:
```

Comment:
Looking at the way this is used, I'd rather not add this to the context. See note on the test below.

Reply:
Didn't have context on that. Thanks for the suggestion